server/services/emailService.ts

```diff
@@ -63,7 +63,7 @@ export default class EmailService {
     userName: string,
     emailAddress: string
   ): Promise<boolean> {
-    const resetURL = `${this.apiRoute}/users/password-reset/r=${confToken}`;
+    const resetURL = `${this.apiRoute}/users/reset/?r=${confToken}`;
```

Comment:
```suggestion
    const resetURL = `${this.apiRoute}/users/password-reset?r=${confToken}`;
```

I think making it more terse here makes sense. We are resetting the password, not the user